exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/metric_exporter/__init__.py

```diff
@@ -368,81 +227,6 @@ def force_flush(self, timeout_millis: float = 10_000) -> bool:
         return True
 
 
-def _translate_value(value: Any) -> KeyValue:
-
-    if isinstance(value, bool):
-        any_value = AnyValue(bool_value=value)
-
-    elif isinstance(value, str):
-        any_value = AnyValue(string_value=value)
-
-    elif isinstance(value, int):
-        any_value = AnyValue(int_value=value)
-
-    elif isinstance(value, float):
-        any_value = AnyValue(double_value=value)
-
-    elif isinstance(value, Sequence):
-        any_value = AnyValue(
-            array_value=ArrayValue(values=[_translate_value(v) for v in value])
-        )
-
-    elif isinstance(value, Mapping):
-        any_value = AnyValue(
-            kvlist_value=KeyValueList(
-                values=[
-                    _translate_key_values(str(k), v) for k, v in value.items()
-                ]
-            )
-        )
-
-    else:
-        raise Exception(f"Invalid type {type(value)} of value {value}")
-
-    return any_value
-
-
-def _translate_key_values(key: str, value: Any) -> KeyValue:
-    return KeyValue(key=key, value=_translate_value(value))
-
-
-def get_resource_data(
```

Comment:
This is the breaking change, I'm unsure if we should maintain the functionality

Reply:
This same function is also duplicated in`opentelemetry.exporter.otlp.proto.grpc.exporter.get_resource_data` but now goes unused

Reply:
Probably would just treat this similarly as the other public functions. Add a deprecation message and suggest users to use a different method. The other alternative is to keep the function as is and continue letting users use it as part of our api.